ui/packages/app/src/validation/validation.js

```diff
@@ -23,3 +23,12 @@ export const validateEmail = email => {
   const expression = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
   return expression.test(email);
 };
+
+// See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set for full details
+export const isValidK8sLabelValue = value => {
```

Comment:
Ok apprently label keys also can have _ in them. If merging them is preferred i'll patch the isDNS1123Label function, probably name it something else since DNS1123 doesn't allow underscores

Reply:
yea i thnik both sounded like its used for the same purpose and validation